package.json

```diff
@@ -13,6 +13,6 @@
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack",
-    "dev": "webpack serve --content-base public"
+    "dev": "webpack serve --content-base public --open"
```

Comment:
this is probably a matter of user preference. I could imagine users preferring it not be present as well. We don't have this in the Rollup or SvelteKit templates, so it's probably better to leave this out